freemocap/core_processes/detecting_things_in_2d_images/mediapipe_stuff/convert_mediapipe_npy_to_csv.py

```diff
@@ -98,6 +98,7 @@ def convert_mediapipe_npy_to_csv(
 
     # %%
     # save broken up npy files
+    # TODO: save `.npy` files as an independent function
```

Comment:
This TODO is incorrect - The `npy` files being fed in here (and the one you're spoofing in the single camera diverter) contain ALL of the mediapipe data (body + hands + face). 

The ones being saved here are individual files for the body, hands, and face.